conceptual/Npgsql/toc.md

```diff
@@ -5,7 +5,7 @@
 ## [3.2](release-notes/3.2.md)
 ## [3.1](release-notes/3.1.md)
 ## [3.0](release-notes/3.0.md)
-# [FAQ](faq.md)
+# [Installation](installation.md)
 # [Compatibility Notes](compatibility.md)
```

Comment:
Makes sense to swap installation and compatibility notes links to move the last one to release notes.

Reply:
I swapped them, but I'm not sure about moving compatibility notes inside the release notes... Seems like it should be visible on the top-level menu no? We can always change later.